SSA/Core/Framework.lean

```diff
@@ -1315,7 +1330,7 @@ theorem Lets.denote_getPureExprAux [LawfulMonad d.m] {Γ₁ Γ₂ : Ctxt d.Ty} {
         Valuation.snoc_last]
       cases eff <;> simp
 
-theorem Lets.denote_getExpr [LawfulMonad d.m] {Γ₁ Γ₂ : Ctxt d.Ty}
+theorem Lets.denote_getExpr [TyDenote d.Ty] [Monad d.m] [DialectDenote d] [LawfulMonad d.m] {Γ₁ Γ₂ : Ctxt d.Ty}
```

Comment:
This being the last theorem with these typeclasses would make a great end of the `section` that has these typeclasses, no?